.chloggen/fix_vcenter-datastore-attributes.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: vcenterreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Removes `vcenter.cluster.name` attribute from Datastores"
```

```suggestion
note: "Removes `vcenter.cluster.name` attribute from `vcenter.datastore` metrics"
```
Nit: Feel free to drop or reword, just a suggestion.